### ISSUE TITLE
Errors in the theme

### ISSUE THREAD

**Issue by Thanatermesis** [godlyranchdressing/retrowave-theme#2]
Congrats for the theme, it is amazing! In fact I just packaged it (tell me if you want a debian-compatible package link of it) and also made this video: https://www.youtube.com/watch?v=OwdcOujtujc

By other side, there's a few errors in the theme that shows up when running some applications, for example **thunar**:
```
(thunar:17391): Gtk-WARNING **: 18:23:16.360: Theme parsing error: gtk.css:17:25: negative values are not allowed.

(thunar:17391): Gtk-WARNING **: 18:23:16.360: Theme parsing error: gtk.css:34:16: '-gtk-icon-size' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.360: Theme parsing error: gtk.css:37:16: '-gtk-icon-size' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.360: Theme parsing error: gtk.css:40:16: '-gtk-icon-size' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.360: Theme parsing error: gtk.css:49:18: '-gtk-icon-filter' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.364: Theme parsing error: gtk.css:594:20: '-gtk-icon-filter' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.364: Theme parsing error: gtk.css:617:20: '-gtk-icon-filter' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.371: Theme parsing error: gtk.css:2043:24: '-gtk-icon-filter' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.375: Theme parsing error: gtk.css:3475:16: '-gtk-icon-size' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.375: Theme parsing error: gtk.css:3486:16: '-gtk-icon-size' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.384: Theme parsing error: gtk.css:5343:20: '-gtk-icon-filter' is not a valid property name

(thunar:17391): Gtk-WARNING **: 18:23:16.386: Theme parsing error: gtk.css:5816:13: Missing opening bracket in color definition

(thunar:17391): Gtk-WARNING **: 18:23:16.386: Theme parsing error: gtk.css:5822:13: Missing opening bracket in color definition

```

or from **lxappearance**:

```
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:363: Unable to locate image file in pixmap_path: "assets/radio-mixed-hover.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:365: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1553: Unable to locate image file in pixmap_path: "assets/checkbox.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1555: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1561: Unable to locate image file in pixmap_path: "assets/checkbox-hover.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1563: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1569: Unable to locate image file in pixmap_path: "assets/checkbox.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1571: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1577: Unable to locate image file in pixmap_path: "assets/checkbox-insensitive.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1579: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1625: Unable to locate image file in pixmap_path: "assets/checkbox-mixed-hover.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1627: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1653: Unable to locate image file in pixmap_path: "assets/radio.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1655: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1661: Unable to locate image file in pixmap_path: "assets/radio-hover.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1663: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1669: Unable to locate image file in pixmap_path: "assets/radio.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1671: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1677: Unable to locate image file in pixmap_path: "assets/radio-insensitive.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1679: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1693: Unable to locate image file in pixmap_path: "assets/radio-checked-hover.png"
/usr/share/themes/retrowave-glow/gtk-2.0/main.rc:1695: Overlay image options specified without filename
/usr/share/themes/retrowave-glow/gtk-2.0/hacks-dark.rc:8: error: invalid identifier 'insensitive_bg_color', expected valid identifier
```

_Note: Unfortunately due to newer dependencies probably I cannot compile (package) the theme from git and I use only the published versions instead_

**Comment by godlyranchdressing** [comment]
Oh, awesome. Thanks!  I'll see about the errors.

**Comment by Thanatermesis** [comment]
Thank you :)

Can you update the versions on https://www.pling.com/p/1391876  ? I can only package it manually for the released (compile) file, so trying to compile it gives me this error:

```
[18:10:25] Local gulp not found in /mkdeb/build/amd64/gnome-themes-retrowave/gnome-themes-retrowave
[18:10:25] Try running: npm install gulp
[18:10:25] Using globally installed gulp
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'gulp-sass'

```

**Comment by godlyranchdressing** [comment]
@Thanatermesis just published it

**Comment by Thanatermesis** [comment]
woops, just another line found:

> (baobab:1504): Gtk-WARNING **: 09:52:55.279: Theme parsing error: gtk.css:17:25: negative values are not allowed.

Thanks for the publish, I updated the package and it will be directly included in the next build of Elive 3.8.14 :)
